backend/services/esweService.js

```diff
@@ -0,0 +1,45 @@
+const { app } = require('../config')
+
+/**
+ * Education skills and work experience (ESWE)
+ */
+class EsweService {
```

Comment:
Any reason why this starts with a capital letter? We generally start with lowercase letters in this code base

Reply:
We have agreed that TypeScript is the way forward, and this is a step in that direction.